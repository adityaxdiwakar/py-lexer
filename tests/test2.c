#include "stdio.h"

int main() {
    float a = 10.0;
    float b = 4.0;
    return a / b;
}

/*
DIRECTIVE
STR(stdio.h)
INT_WORD
IDENT(main)
LBRACK

FLOAT_WORD
IDENT(a)
EQUALS
FLOAT
SEMI

FLOAT_WORD
IDENT(b)
EQUALS
FLOAT
SEMI

RETURN
IDENT(a)
DIV
IDENT(b)
SEMI
RBRACK
*/