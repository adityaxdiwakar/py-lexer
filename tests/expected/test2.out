DIRECTIVE(include)
STR(stdio.h)
INT_WORD
IDENT(main)
LPAREN
RPAREN
LBRACK
FLOAT_WORD
IDENT(a)
EQUALS
FLOAT(10.0)
SEMI
FLOAT_WORD
IDENT(b)
EQUALS
FLOAT(4.0)
SEMI
RETURN
IDENT(a)
DIV
IDENT(b)
SEMI
RBRACK